src/pages/About/AboutKyberSwap/index.tsx

```diff
@@ -168,7 +169,7 @@ export const KSStatistic = () => {
         <ForTraderInfoRow>
           <ForTraderInfoCell>
             <Text fontWeight="600" fontSize="24px">
-              {MAINNET_NETWORKS.length - 1}+
+              {MAINNET_NETWORKS.length}+
```

Comment:
Why

Reply:
why length -1?

Reply:
Because it's correct. If we support 14 chain, we can show that it's 13+ :trollface: not 14+

Reply:
14+ mean more is coming